### ISSUE TITLE
Set task name for different tasks

### ISSUE THREAD

**Issue by LavendaRaphael** [deepmodeling/dpdispatcher#315]
- If not set, the task name is the filename, i.e. a long hash string.
- If the task name is set in the resources, all jobs will be the same task name.

So is there a way to set a specific name for each task? It is useful for monitoring tasks.

**Comment by njzjz** [comment]
> * If not set, the task name is the filename, i.e. a long hash string.

Do you mean the "job" name? A job may contain multiple tasks.

**Comment by LavendaRaphael** [comment]
> > * If not set, the task name is the filename, i.e. a long hash string.
> 
> Do you mean the "job" name? A job may contain multiple tasks.

Yes, if the job name is set, all the contained tasks will have the same name.
